### ISSUE TITLE
bug(progress-spinner ): fix primary disabled button color contrast issue

### ISSUE THREAD

**Issue by ArtBlue** [eBay/skin#2010]
Fix color contrast violation on Button > Primary > Disabled...

![image](https://user-images.githubusercontent.com/1675667/225401390-877bf34c-f6ee-4084-8863-5ce12eeee469.png)

**Comment by ArtBlue** [comment]
Randy to provide the colors to use to resolve the issue.

**Comment by randybascue** [comment]
Why is there a spinner on a disabled button though? Can't we keep it primary blue and add the spinner instead?

**Comment by randybascue** [comment]
![image](https://user-images.githubusercontent.com/35155833/225417892-7f19ff84-82c3-45fa-81d4-5d1c55277d9b.png)

**Comment by randybascue** [comment]
[Figma reference](https://www.figma.com/file/zEBdEhbonrBOGzZ0fXzWvM/eBay-Design-System?node-id=65849%3A91206&t=j92KgiHHsFYvjyKH-11)

**Comment by ianmcburnie** [comment]
In skin, we could set the button background to stay as blue when loading button has disabled attribute. I don't think we really need the grey background in that case - it should be pretty apparent from the spinner that the button is not clickable in this state.
